keyboards/handwired/bongo80/keymaps/default/keymap.c

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2023 Sam Kelly (@samkellu)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "bongo80.h"
+
+enum layers {
+	_QWERTY, _MEDIA
+};
+
```

Comment:
Quick tip: 
If you add a comma at the end of the last entry, clang-format won't reformat it to be a single line. 

Also, this is a bit more readable:
```suggestion
enum layers {
	_QWERTY, 
	_MEDIA,
};

```

Reply:
Good to know, thanks!